content/development/guides/tutorials.en.yaml

```diff
@@ -47,12 +47,12 @@
   key: Kotti Solidity ETC Oh My
   title: Kotti, Solidity, ETC, Oh My!
   link: https://medium.com/ethereum-classic/kotti-solidity-etc-oh-my-2ae36926454d
-  text: This guide illustrates an initital configure for Ethereum Classic's Kotti testnet and a specifc version of Solidity for ETC<>ETH compatible daap development. Note: Kotti is the recommended testnet for dapp developers. Mordor is recommended for protocol level testing.
+  text: This guide illustrates an initital configure for Ethereum Classic's Kotti testnet and a specifc version of Solidity for ETC<>ETH compatible daap development. Note, Kotti is the recommended testnet for dapp developers. Mordor is recommended for protocol level testing.
 - 
   key: How to contribute to Mordor Testnet
   title: How to contribute to Mordor Testnet
   link: https://core-geth.org/contribute-to-mordor-testnet
-  text: This guide will teach you how to mine testnet ETC (mETC) and run your own local block explorer with Expedition. Note: Kotti is the recommended testnet for dapp developers. Mordor is recommended for protocol level testing. 
+  text: This guide will teach you how to mine testnet ETC (mETC) and run your own local block explorer with Expedition. Note, Kotti is the recommended testnet for dapp developers. Mordor is recommended for protocol level testing. 
```

Comment:
If you want to keep the colon, simply wrap the line in double quotes, so this would be 

```
  text: "This guide will teach you how to mine testnet ETC (mETC) and run your own local block explorer with Expedition. Note: Kotti is the recommended testnet for dapp developers. Mordor is recommended for protocol level testing."
```